macros/src/test/scala/MacroCompilerSpec.scala

```diff
@@ -335,7 +335,7 @@ s"""
      * @param readEnable Has a read enable port?
      * @param mask Mask granularity (# bits) of the port or None. */
     def generateReadWriteFooterPort(prefix: String, readEnable: Boolean, mask: Option[Int]): String = {
-      generatePort(libPortPrefix, lib_addr_width, libWidth,
+      generatePort(prefix, lib_addr_width, libWidth,
```

Comment:
I think you should make the default argument for these prefixes the same as it was before.

Reply:
This specific change actually fixes a bug which prevented two of the unit tests from passing. I think the caller's arguments didn't change, so are there any other changes to make?

Reply:
Ah I didn't read far enough up in the diff. Yeah this is fine.